docs/video-miners/core-concepts/roles-and-responsibilities.md

```diff
@@ -4,55 +4,73 @@ title: Roles and Responsibilities
+
+The quality of transcoding services is influenced by a combination of factors, including but not limited to: 
+
+- Economic security (the stake of the orchestrator and its delegators which can be slashed if the work is performed incorrectly), 
+
+- Geographic coverage and transcoding capacity (affected by bandwidth and computational capacity).
+
+Since, based on these factors, a broadcaster selects orchestrators to send work to, an orchestrator's earning potential depends on the ability to perform well in each of these areas.
+
+An orchestrator can also be a transcoder, with the additional option of outsourcing transcoding to one or many separate transcoders; similar to how PoW mining pools outsource PoW hashing to one or many separate miners. Orchestrators outsourcing transcoding to separate transcoders are commonly referred to as
 [transcoding pools](/video-miners/core-concepts/pools).
 
-## Transcoding
+### Running an Orchestrator
+
+Video miners who run an orchestrator:
+
+- Earn LPT rewards
+- Participate in governance with your staked LPT
+- Can scale up your operation over time to maximize the amount of work received (instead of only sharing work received by another orchestrator -- like a transcoder)
+- Should be able to maintain a static IP or DNS hostname for broadcasters to access
+- Should be able to maintain a publicly accessible server on the Internet
+- Manage and secure Ethereum private keys
+- Manage and spend ETH to pay for transactions to redeem payments and distribute rewards
+
+### About Transcoders
+
+Transcoders perform work on the network with their hardware resources, providing computational capacity to orchestrators. 
+
+Transcoding is the process of taking an input source video in one format and bitrate, converting it to many output rendition videos with different formats and bitrates such that the video can be played on the majority of devices running video at any internet connection speed. Additional resources about transcoding can be found in [the Livepeer Blog](https://livepeer.com/blog/intro-to-transcoding).
+
+### Transcoding on the Livepeer Network
+
+In the Livepeer network, broadcasters and video miners establish a price for transcoding on the open market. Exact economics for transcoders depends entirely on the implementation of the [trancoding pool](/video-miners/core-concepts/pools) they belong to. For example, one pool could distribute [LPT rewards and ETH fees](/video-miners/core-concepts/earnings#fees) to transcoders, while others just distribute ETH fees. 
+
+Once a broadcaster has selected one or many video miners to utilize, it will send segments of a video stream to the video
+miners with micropayments based on an agreed-upon price. The video miners then return transcoded results for each segment to the broadcasters.
```

Comment:
okay so just one clarification here - broadcasters interact with *orchestratrors* rather than transcoders. The orchestrator is the middleman and someone running a standalone transcoder will not receive work because no one's doing the orchestrator job for them

Reply:
so how does the transcoding get distributed?